frontend/src/features/contact/list/ContactSearchResults/ContactSearchResults.tsx

```diff
@@ -0,0 +1,65 @@
+import { Table } from 'components/Table';
+import { SortDirection, TableSort } from 'components/Table/TableSort';
+import { IContactSearchResult } from 'interfaces';
+import { useCallback } from 'react';
+
+import columns from './columns';
+
+export interface IContactSearchResultsProps {
+  results: IContactSearchResult[];
+  pageCount?: number;
+  pageSize?: number;
+  pageIndex?: number;
+  sort?: TableSort<IContactSearchResult>;
+  setSort?: (value: TableSort<IContactSearchResult>) => void;
+  setPageSize?: (value: number) => void;
+  setPageIndex?: (value: number) => void;
+  loading?: boolean;
+}
+
+/**
+ * Display a react table with the results filtered by the {@link ContactFilter}
+ * @param {IContactSearchResult} props
+ */
+export function ContactSearchResults(props: IContactSearchResultsProps) {
+  const { results, sort = {}, setSort, setPageSize, setPageIndex, ...rest } = props;
+
+  // results sort handler
+  const handleSortChange = useCallback(
+    (column: string, nextSortDirection: SortDirection) => {
+      if (!setSort) return null;
```

Comment:
nit: Thoughts on brackets on every if/else? I saw some that have brackets and others that do not.

Reply:
I'm one of those people that don't think brackets/no brackets affect readability.

Reply:
I really like consistency :)

Reply:
I think we've talked about this and almost always use brackets except for one-liners like this case. 
If team thinks otherwise we can change it